sdk/resourcemanager/azure-resourcemanager-resources/src/main/java/com/azure/resourcemanager/resources/fluentcore/utils/ResourceManagerThrottlingInfo.java

```diff
@@ -83,7 +91,7 @@ public static ResourceManagerThrottlingInfo fromHeaders(HttpHeaders headers) {
 
     /**
      * refer https://docs.microsoft.com/en-us/azure/virtual-machines/troubleshooting/troubleshooting-throttling-errors
-     * @return a specific rate limit from compute
+     * @return a specific rate limit header value without parsed from compute
```

Comment:
`a specific rate limit header value from compute` should be enough. `without parsed` here seems grammatically not correct.

Reply:
done